main.js

```diff
@@ -1,39 +1,88 @@
 var activityButtons = document.querySelector(".activity-buttons");
 var startActivityBtn = document.querySelector(".start-activity-button");
-var timeInput = document.querySelector(".time-input");
+var timeInput = document.querySelector(".time-input"); // refactor???
+var description = document.querySelector(".description-input");
+var minutes = document.querySelector(".minutes-input");
+var seconds = document.querySelector(".seconds-input");
 var currentActivity;
```

Comment:
I wonder if there's a way to put just querySelectors in their own function that we can invoke when we need to use them?  Not sure if that's possible, but I see that in the rubric it is suggested that it's possible to only have the two global variables.